wiki/markdown/configuration.md

```diff
@@ -168,4 +168,57 @@ and the settings are defined in the **ldap.server.list** property.
             "password": "",\
             "user.filter": "uid={0}"\
         }\
-    ]\
\ No newline at end of file
+    ]\
+
+## Password encryption
+
+In a production environment and filesystems administrator most likely don't want to store passwords in clear text.
```

Comment:
you can add a "," between "filesystems" and "administrator", it reads wierdly otherwise :)

Reply:
Done.